src/components/PostList/PostList.jsx

```diff
@@ -1 +1,12 @@
-export const PostList = () => <>Put the list here</>;
+import { PostInfo } from '../PostInfo/PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
+    {' '}
+    <div className="PostList">
+      {posts.map(post => {
+        return <PostInfo post={post} />;
```

Comment:
You should write attribute key when you use method map on posts. Post has id key which is uniq